themes/alacritty_0_12.toml

```diff
@@ -0,0 +1,44 @@
+# Alacritty's default color scheme pre-0.13 (based on tomorrow_night)
+# https://github.com/alacritty/alacritty/blob/v0.12.3/alacritty/src/config/color.rs
+
+[colors.bright]
+black = "#666666"
+red = "#d54e53"
+green = "#b9ca4a"
+yellow = "#e7c547"
+blue = "#7aa6da"
+magenta = "#c397d8"
+cyan = "#70c0b1"
+white = "#eaeaea"
+
+[colors.dim]
+black = "#131415"
+red = "#864343"
+green = "#777c44"
+yellow = "#9e824c"
+blue = "#556a7d"
+magenta = "#75617b"
+cyan = "#5b7d78"
+white = "#828482"
+
+[colors.hints]
+start = { foreground = "#1d1f21", background = "#e9ff5e" }
+end = { foreground = "#e9ff5e", background = "#1d1f21" }
+
+[colors.normal]
```

Comment:
Could we have normal above bright, the order is usually `normal`, `bright`, and then `dim`.

Reply:
Done. I had the sections sorted alphabetically previously.